pex/bin/pex.py

```diff
@@ -523,8 +551,9 @@ def main():
 
     log('Running PEX file at %s with args %s' % (pex_builder.path(), cmdline), v=options.verbosity)
     pex = PEX(pex_builder.path(), interpreter=pex_builder.interpreter)
-    sys.exit(pex.run(args=list(cmdline)))
+    raise SystemExit(pex.run(args=list(cmdline)))
```

Comment:
this change still feels weird to me - and I think to others it will also. since the two are fundamentally identical, why not just stick with the more intuitive/common form of `sys.exit`?

fwiw, you shouldn't need to explicitly raise here to be able to handle SystemExit in your tests:

```
>>> try:
...     sys.exit(1)
... except SystemExit:
...     print('not exiting!')
...     
... 
not exiting!
```